tools/depends/configure.ac

```diff
@@ -270,6 +285,7 @@ case $host in
     optimize_flags="-O2"
     platform_cxxflags="$platform_cflags"
     platform_os="linux"
+    ffmpeg_options="--enable-vaapi --enable-vdpau --cpu=$use_cpu"
```

Comment:
this overrides the user passed values.
I suggest naming this ffmpeg_options_default and setting them further down in case --with-ffmpeg-options was not set.
Same for the other cases